test/stat_tracker_test.rb

```diff
@@ -37,22 +39,165 @@ def test_it_can_return_the_highest_total_score
+    assert_equal expected, @stat_tracker.total_wins_by_team
+  end
+
+  def test_it_can_show_total_loss
+    expected = {3=>2, 6=>0, 5=>0, 20=>1, 19=>0, 7=>1, 52=>0, 10=>1, 26=>0}
+    assert_equal expected, @stat_tracker.total_loss_by_team
+  end
+
+  def test_it_can_show_total_ties
+    expected = {3=>1, 6=>0, 9=>1, 8=>1, 5=>2, 20=>0, 19=>0,
+      7=>0, 52=>0, 10=>0, 26=>0, 22=>1}
+    assert_equal expected, @stat_tracker.total_tie_by_team
   end
+
+  def test_it_can_display_home_or_away_games_by_team
+    expected_home = {6=>1, 8=>1, 5=>4, 19=>1, 52=>1, 26=>1}
+
+    assert_equal expected_home, @stat_tracker.hoa_games_by_team("home")
+
+    expected_away = {3=>3, 9=>1, 6=>1, 20=>1, 7=>1, 10=>1, 22=>1}
+    assert_equal expected_away, @stat_tracker.hoa_games_by_team("away")
+  end
+
+  def test_it_can_display_all_goals_scored_hoa_by_team
+    expected_home = {3=>0, 6=>3, 9=>0, 8=>2, 5=>7, 20=>0, 19=>2,
+      7=>0, 52=>2, 10=>0, 26=>2, 22=>0}
+    assert_equal expected_home, @stat_tracker.hoa_goals_by_team("home")
+
+    expected_away = {3=>6, 6=>3, 9=>2, 8=>0, 5=>0, 20=>1,
+      19=>0, 7=>1, 52=>0, 10=>1, 26=>0, 22=>2}
+
+    assert_equal expected_away, @stat_tracker.hoa_goals_by_team("away")
+  end
+
+  def test_it_can_display_home_or_away_wins_by_team
+    expected_home = {3=>0, 6=>1, 9=>0, 8=>0, 5=>1, 20=>0, 19=>1,
+      7=>0, 52=>1, 10=>0, 26=>1, 22=>0}
+    assert_equal expected_home, @stat_tracker.hoa_wins_by_team("home")
+
+    expected_away = {3=>0, 6=>1, 9=>0, 8=>0, 5=>0, 20=>0, 19=>0,
+      7=>0, 52=>0, 10=>0, 26=>0, 22=>0}
+
+    assert_equal expected_away, @stat_tracker.hoa_wins_by_team("away")
+  end
+
+  def test_it_can_display_home_or_away_losses_by_team
+    expected_home = {3=>0, 6=>0, 9=>0, 8=>0, 5=>1, 20=>0,
+      19=>0, 7=>0, 52=>0, 10=>0, 26=>0, 22=>0}
+    assert_equal expected_home, @stat_tracker.hoa_loss_by_team("home")
+
+    expected_away = {3=>2, 6=>0, 9=>0, 8=>0, 5=>0, 20=>1, 19=>0, 7=>1,
+      52=>0, 10=>1, 26=>0, 22=>0}
+    assert_equal expected_away, @stat_tracker.hoa_loss_by_team("away")
+  end
+
+  def test_it_can_display_home_or_away_ties_by_team
+    expected_home = {3=>0, 6=>0, 9=>0, 8=>1, 5=>2, 20=>0, 19=>0,
+      7=>0, 52=>0, 10=>0, 26=>0, 22=>0}
+    assert_equal expected_home, @stat_tracker.hoa_tie_by_team("home")
+
+    expected_away = {3=>1, 6=>0, 9=>1, 8=>0, 5=>0, 20=>0, 19=>0,
+      7=>0, 52=>0, 10=>0, 26=>0, 22=>1}
+    assert_equal expected_away, @stat_tracker.hoa_tie_by_team("away")
+  end
+
+  ### Seasonal Stats Below
+  def test_it_knows_all_the_game_ids_in_a_given_season
+    assert_equal [2012030221, 2012030121, 2012030311, 2012020701, 2012020587], @stat_tracker.game_ids_in_season(20122013)
+  end
+
+  def test_it_knows_the_number_of_games_played_by_teams_in_a_season
+    expected = {3 => 2, 5 => 3, 10 => 1, 26 => 1, 22 => 1}
+    assert_equal expected, @stat_tracker.games_by_team_by_season(20152016)
+  end
+
+  def test_it_knows_coachs_by_season
+    expected_1213 = {
```

Comment:
Can we change this to expected only? Want to be sure we don't lose track of this

Reply:
Should be fixed now, I had made some changes but forgot to commit them before pushing this branch.  Good catch!